pom.xml

```diff
@@ -413,8 +413,15 @@
 				<groupId>org.openmrs.module</groupId>
 				<artifactId>webservices.rest-omod-common</artifactId>
 				<version>${webservices.restModuleVersion}</version>
-				<scope>test</scope>
+				<scope>provided</scope>
 			</dependency>
+			<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
+			<version>${webservices.restModuleVersion}</version>
+			<scope>provided</scope>
+			<type>test-jar</type>
+    	</dependency>
```

Comment:
@reagan-meant update this file to the latest version from upstream master

Reply:
Done